lib/generators/sufia/install_generator.rb

```diff
@@ -172,8 +172,8 @@ def inject_routes
         "  # This behavior seems to show up only in production mode.\n" \
         "  mount Sufia::Engine => '/'\n"
 
-      sentinel = /devise_for :users/
-      inject_into_file 'config/routes.rb', routing_code, after: sentinel, verbose: false
+      sentinel = /end\Z/
```

Comment:
This approach will break if folks have a malformatted routes file, say if another gem sticks an `end` statement at the beginning of a line before the end of the file, but meh, I'm not sure there's a better approach.